build.gradle

```diff
@@ -59,7 +59,7 @@ buildscript {
     repositories {
         maven { url 'https://maven.pkg.jetbrains.space/kotlin/p/dokka/dev' }
         // kotlin-dev with space redirector
-        maven { url "https://cache-redirector.jetbrains.com/maven.pkg.jetbrains.space/kotlin/p/kotlin/dev" }
```

Comment:
This repository is required for TC aggregate and nightly builds, do not remove it.

Reply:
Yes, already fixed.